toolchain/diagnostics/diagnostic_emitter.h

```diff
@@ -5,6 +5,7 @@
 #ifndef CARBON_TOOLCHAIN_DIAGNOSTICS_DIAGNOSTIC_EMITTER_H_
 #define CARBON_TOOLCHAIN_DIAGNOSTICS_DIAGNOSTIC_EMITTER_H_
 
+#include <concepts>
```

Comment:
Was this already used here, or is it leftover from intermediate state? (not examining full file)

```suggestion
```

Reply:
Another clangd included, my bad